string person_name
sensor_msgs/Image image
---
string error_msg
